dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -94,11 +153,13 @@ public boolean equals(@Nullable Object obj) {
     }
 
     @Override
+    @Pure
     public int hashCode() {
         return Objects.hash(constructor, arguments);
```

Comment:
Add the new field, in `toString`, `equals`, `hashCode`, and `getOperands`!

Reply:
Thanks!